packages/plugins/typescript/react-query/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @graphql-codegen/typescript-react-query
 
+## 3.7.0-alpha-1370c16b0.0
+
+### Minor Changes
+
+- 1370c16b0: Add LegacyMode option (Set this to false, if you using `@tanstack/react-query`)
+
```

Comment:
we don't need to manually update the changelog. Adding yarn changeset will take care of it
```suggestion
```